specification/maps/data-plane/DataRegistry/preview/2022-12-01-preview/dataregistry.json

```diff
@@ -175,22 +175,22 @@
             "headers": {
               "x-ms-error-code": {
                 "type": "string",
-                "description": "Error code for specific error that occurred."
+                "description": "Error code of the error that occurred."
               }
             },
             "x-ms-error-response": true
           }
         }
       },
       "put": {
-        "description": "**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\nThe endpoint of Data Registry APIs is bound to the region of your maps account. Data Registry Service is not available for global maps account.\n\nThe Data Registry Register API allows the caller to register their data content uploaded to Azure Storage Account in the Azure Maps Service with customer provided udid, restricted in GUID format; or to replace an existing udid with another data registry.\n\nYou can use this API in scenarios like registering a collection of Geofences in `GeoJSON` format for use in [Azure Maps Geofencing Service](https://docs.microsoft.com/rest/api/maps/spatial), or registering a set of DWG design files as a zipped [Drawing Package](https://docs.microsoft.com/en-us/azure/azure-maps/drawing-requirements) for use in [Azure Maps Creator Conversion Service](https://docs.microsoft.com/en-us/rest/api/maps/v2/conversion).\n\n## Submit Register or Replace Request\n\nTo register or replace your data registry you will use a `PUT` request.\n\nFor example, to register a collection of geofences  in `GeoJSON` format, set Azure blob data registry object representing how Azure Maps accesses your geojson file. and `dataFormat` to _geojson_ in the request body.\n\nThe Data Registry Register or Replace API performs a [long-running request](https://aka.ms/am-creator-lrt-v2).",
```

Comment:
Please add some description about the data size limitation.
The maximum data size customer can register is 1 GiB.

Reply:
Does this work?

`The maximum file size that can be registered is one gigabyte.`

Reply:
Yes, it works. thank you!